test/configure_packages.jl

```diff
@@ -0,0 +1,26 @@
+using Pkg
+Pkg.activate(dirname(@__DIR__))
+Pkg.instantiate()
+
+# Configure the test setup based on environment variables set in CI.
+# First, we get the settings and remove all local preference configurations
+# that may still exist.
+const JULIA_HDF5_PATH = get(ENV, "JULIA_HDF5_PATH", "")
+rm(joinpath(dirname(@__DIR__), "LocalPreferences.toml"); force=true)
+
+# Next, we configure MPI.jl appropriately.
+import MPIPreferences
+MPIPreferences.use_system_binary()
+
+# Finally, we configure HDF5.jl as desired.
+import UUIDs, Preferences
+Preferences.set_preferences!(
+    UUIDs.UUID("f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"), # UUID of HDF5.jl
+    "libhdf5" => joinpath(JULIA_HDF5_PATH, "libhdf5.so");
```

Comment:
Could libhdf5.so also be in a `lib` or `lib64` subdirectory?

Reply:
Not as it is now. Primarily, I created this file for CI, where we know, where the binaries are located. Now, that we suggest the user to have a look at this file, it would probably be nice to also allow the binaries to be in `joinpath(JULIA_HDF5_PATH, "lib")` or `joinpath(JULIA_HDF5_PATH, "lib64")`.